nanome/_internal/_structure/_residue.py

```diff
@@ -97,6 +97,7 @@ def _shallow_copy(self, conformer_number = None):
         residue._ribbon_color = self._ribbon_color.copy()
         residue._labeled = self._labeled 
         residue._label_text = self._label_text 
+        residue._ignored_alt_locs = self._ignored_alt_locs 
```

Comment:
```suggestion
        residue._ignored_alt_locs = self._ignored_alt_locs[:]
```
copy list instead of copying reference